hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -293,7 +293,7 @@ def process_pol(in_file, rtc_name, out_name, pol, res, look_fact, match_flag, de
     if area:
         shutil.move("{}.flat.tif".format(out_name), "{}/{}_flat_{}.tif".format(out_dir, out_name, pol))
 
-    os.chdir("..")
+    os.chdir("../..")
```

Comment:
I don't see a corresponding that makes this necessary?  What's the reasoning here?

Reply:
Woops, looks like it came in dce3a6c6499dc74b3058a28e3af0f19a533947bf and was an artifact of an auto-refactor; backing it out (and on line 422)